charts/eks/templates/hostpath-mapper-daemonset.yaml

```diff
@@ -0,0 +1,99 @@
+{{- if .Values.hostpathMapper.enabled }}
+apiVersion: apps/v1
+{{- if not .Values.hostpathMapper.dev }}
+kind: DaemonSet
+{{- else  }}
+kind: Deployment
+{{- end }}
+metadata:
+  name: {{ .Release.Name }}
```

Comment:
Doesn't just release name cause issues with the regular vcluster deployment?

Reply:
using `{{ .Release.Name }}-hostpath-mapper` now